app/components/common/cardList/CardList.js

```diff
@@ -34,23 +34,23 @@ export default class CardList extends Component {
   render() {
     return (
       <Card title={this.props.title}>
-        {this.props.list.map(item =>
+        {this.props.list.map(item => (
```

Comment:
PRETTIER CHANGED?

Reply:
I guess.

Reply:
try running prettier on every file to check it

Reply:
prettier indeed